README.md

```diff
@@ -118,13 +123,35 @@ Within this file, you can define the `CREATE INDEX` statements for your bucket,
 ```sql
 CREATE PRIMARY INDEX `#primary` ON default WITH {"defer_build": true};
 CREATE INDEX `Types` ON default (`type`) WITH {"defer_build": true};
-BUILD INDEX ON default (`#primary`, `Types`)
+BUILD INDEX ON default (`#primary`, `Types`);
 ```
 
 ### Creating Indexes with YAML
 
 Alternatively, you may add YAML files with index definitions under the `/startup/<bucketname>/indexes` folder.  This operation uses [couchbase-index-manager](https://www.npmjs.com/package/couchbase-index-manager) to create the indexes.  [See here](https://www.npmjs.com/package/couchbase-index-manager#definition-files) for an explanation of the YAML file format.
 
+### Analytics Dataset Setup
+
+To setup the analytics service datasets, add a directory under the `/startup/<bucketname>/analytics` folder. Within this folder create a text file named `dataset.n1ql`. For example, `/startup/default/analytics/dateset.n1ql`.  Note that the names are case sensitive.
+
+Within this file, you can define the `CREATE DATASET` statements for your bucket, separated by semicolons. **IMPORTANT** Make sure to append the proper `USE` statement to each `CREATE DATASET` statement so that it's placed in the proper DATAVERSE. Additionaly, always end your file with a `CONNECT LINK Local;` statement.
+
+```sql
```

Comment:
The documentation doesn't seem to align with the code/example, which appear to be using JSON files rather than N1QL files.